internals/webpack.config.js

```diff
@@ -16,10 +15,14 @@ module.exports = {
         loader: 'babel-loader',
         exclude: /node_modules/,
         query: {
-          presets: ['latest', 'react'],
-          plugins: ['transform-flow-strip-types']
-        }
-      }
-    ]
-  }
-}
+          presets: ['latest', 'react', 'stage-0', {
+            plugins: [
+              './build/babelRelayPlugin',
+            ],
+          }],
+          plugins: ['transform-flow-strip-types'],
```

Comment:
i think you only need to specify plugins once. the relay and flow plugins can just be in the same array